localisation/wc_culture_buildings.csv

```diff
@@ -1,212 +1,212 @@
+ca_culture_quilboar_3;Grounds of Agamaggan;;�bungspl�tze von Agamaggan;;;;;;;;;;;x
+ca_culture_quilboar_4;Grounds of Agamaggan;;�bungspl�tze von Agamaggan;;;;;;;;;;;x
+ca_culture_quilboar_desc;Though the majority of quilboar society is dominated by shamanism and geomancy, powerful and brave warriors are still very much prized among all communities. Several of them train here in the inspiration that follows from being descendents of Agamaggan.;;Obwohl die Mehrheit der Stacheleber-Gesellschaft von Schamanismus und Geomantie dominiert wird, werden m�chtige und tapfere Krieger in allen St�mmen nach wie vor sehr gesch�tzt. Einige von ihnen trainieren hier in der Inspiration, die sich aus ihrer Abstammung von Agamaggan ergibt.;;;;;;;;;;;x
+ca_culture_pygmy_1;Pygmy Billets;;Pygm�en-Quartier;;;;;;;;;;;x
+ca_culture_pygmy_2;Pygmy Billets;;Pygm�en-Quartier;;;;;;;;;;;x
+ca_culture_pygmy_3;Pygmy Billets;;Pygm�en-Quartier;;;;;;;;;;;x
+ca_culture_pygmy_4;Pygmy Billets;;Pygm�en-Quartier;;;;;;;;;;;x
+ca_culture_pygmy_desc;Hardened by the deserts and primitive jungles of Azeroth, pygmies are small savage humanoids with little reason. Though tiny compared to other races, their warriors are brutal and quick, creating a deadly force able to defend any threatened tribe.;;Abgeh�rtet durch die W�sten und primitiven Dschungel Azeroths sind Pygm�en kleine wilde Humanoide mit wenig Vernunft. Obwohl sie im Vergleich zu anderen Rassen winzig sind, sind ihre Krieger brutal und schnell und bilden eine t�dliche Streitmacht, die in der Lage ist, jeden bedrohten Stamm zu verteidigen.;;;;;;;;;;;x
+ca_culture_orc_1;Wolf Beastiary;;Wolfs-Bestiarium;;;;;;;;;;;x
+ca_culture_orc_2;Wolf Beastiary;;Wolfs-Bestiarium;;;;;;;;;;;x
+ca_culture_orc_3;Wolf Beastiary;;Wolfs-Bestiarium;;;;;;;;;;;x
+ca_culture_orc_4;Wolf Beastiary;;Wolfs-Bestiarium;;;;;;;;;;;x
+ca_culture_orc_desc;From beyond the Dark Portal, the orcs befriended and tamed the wolves of Draenor. With duties such as scouting and companionship, these large and swift canines were chosen as the orcs' favourite method of transportation that so well happened to be suited for war. Later, when the orcs became more belligerent, these wolves were bred specifically for size and stamina so they could bear armoured riders into battle. The ferocity that the wolf and its rider shows in battle, leaves most soldiers trembling in fear.;;Bereits jenseits des dunklen Portals schlossen die Orks Freundschaft mit den W�lfen von Draenor und z�hmten sie. Mit Aufgaben wie Erkundung und Begleitung wurden diese gro�en und schnellen Fleischfresser zum bevorzugten Transportmittel der Orks, das geradezu ideal f�r den Krieg geeignet war. Sp�ter, als die Orks immer kriegerischer wurden, wurden diese W�lfe speziell auf Gr��e und Ausdauer gez�chtet, damit sie gepanzerte Reiter in die Schlacht tragen konnten. Die Grausamkeit, die der Wolf und sein Reiter in der Schlacht zeigen, l�sst die meisten Soldaten vor Angst zittern.;;;;;;;;;;;x
```

Comment:
Changed the last part of the second sentence a little.
```suggestion
ca_culture_orc_desc;From beyond the Dark Portal, the orcs befriended and tamed the wolves of Draenor. With duties such as scouting and companionship, these large and swift canines were chosen as the orcs' favourite method of transportation that were also well suited for war. Later, when the orcs became more belligerent, these wolves were bred specifically for size and stamina so they could bear armoured riders into battle. The ferocity that the wolf and its rider shows in battle, leaves most soldiers trembling in fear.;;Bereits jenseits des dunklen Portals schlossen die Orks Freundschaft mit den W�lfen von Draenor und z�hmten sie. Mit Aufgaben wie Erkundung und Begleitung wurden diese gro�en und schnellen Fleischfresser zum bevorzugten Transportmittel der Orks, das geradezu ideal f�r den Krieg geeignet war. Sp�ter, als die Orks immer kriegerischer wurden, wurden diese W�lfe speziell auf Gr��e und Ausdauer gez�chtet, damit sie gepanzerte Reiter in die Schlacht tragen konnten. Die Grausamkeit, die der Wolf und sein Reiter in der Schlacht zeigen, l�sst die meisten Soldaten vor Angst zittern.;;;;;;;;;;;x
```

Reply:
changed